src/modules/Builder/features/ActivityItemsFlow/ItemFlow/ItemFlowContent/StaticConditionRow/StaticConditionRow.tsx

```diff
@@ -0,0 +1,38 @@
+import { Skeleton, SkeletonProps } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+
+import { StyledFlexTopCenter, StyledTitleMedium } from 'shared/styles';
+
+import { StaticConditionRowProps } from './StaticConditionRow.types';
+
+const commonSxProps = {
```

Comment:
as well, pls move to StaticConditionRow.const.ts

Reply:
moved to styles